app/src/main/java/tech/relaycorp/gateway/data/disk/SensitiveStore.kt

```diff
@@ -24,6 +24,7 @@ class SensitiveStore
 
     suspend fun store(location: String, data: ByteArray) {
         withContext(Dispatchers.IO) {
+            buildFile(location).delete()
```

Comment:
Does this work when the file doesn't exist yet? Can you please write a test for this?